### PR TITLE
Fixing english original text

### PR DESCRIPTION
**Pull request by sheilagomes**
Changed English original text as requested by Yuli Verena Monarin - LOC5806

### CODE REVIEW

**Review by camilavcoutinho — APPROVED** [review]

File: messages/en.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
-  "profile-form.field.corporateDocument": "Coporate document",
-  "profile-form.field.corporateName": "Corporate name",
-  "profile-form.field.stateRegistration": "State registration",
-  "profile-form.field.tradeName": "Trade name",
+  "profile-form.field.birthDate": "Birth Date",
+  "profile-form.field.homePhone": "Phone Number",
+  "profile-form.field.businessPhone": "Corporate Phone",
+  "profile-form.field.corporateDocument": "Corporate Document",
+  "profile-form.field.corporateName": "Corporate Name",
+  "profile-form.field.stateRegistration": "State Registration",
```
Comment (salesfelipe) [inline messages/en.json:17]: Is there a reason for this change?

Comment (sheilagomes) [reply messages/en.json:17]: In English we usually capitalize all the words in a title, and, in this case, these are the titles for the fields, so they should be capitalized. Also, some typos were corrected.

Comment (salesfelipe) [reply messages/en.json:17]: Ok, but this should be done for the other branch aswell

Comment (sheilagomes) [reply messages/en.json:17]: All right, thank you for the heads-up! I'll do that.

Comment (sheilagomes) [reply messages/en.json:17]: Done, @salesfelipe ! The other PR is [here](https://github.com/vtex/profile-form/pull/129).

**Review by arthurepc — APPROVED** [review]
LGTM

**Pull request merged**